src/Components/ClustersListTable/ClustersListTable.js

```diff
@@ -137,23 +145,32 @@ const ClustersListTable = ({
     }
   }, [filters, filterBuilding]);
 
-  const buildFilteredRows = async (items) => {
-    setRowsFiltered(false);
+  const buildFilteredRows = async (items, signal) => {
     const filtered = items.filter((it) => {
       return passFiltersCluster(it, filters);
     });
     const clusterArr = filtered.map((cluster) => cluster.cluster_id);
     let upgradeArr = [];
     if (clusterArr.length > 0) {
       try {
-        const upgradeRisks = await getClustersUpgradeRisks({
-          clusters: clusterArr,
-        }).unwrap();
-        if (upgradeRisks.status === 'ok') {
-          upgradeArr = upgradeRisks.predictions;
+        const res = await axios.post(
```

Comment:
Let's use the shared useAxiosWithPlatformInterceptors for better error handling. Refer: https://redhat-internal.slack.com/archives/C02T90PLJ5V/p1711451779574089.

Reply:
I'm unable to cancel the request with this hook, looks like its a different instance on each request. I've tried to use the hook and importing the `instance` from `/interceptors` as well.